Warn `kyma-project.io` users about early stage of the cli (#2381)
kyma-project/cli: docs/user/README.md

# Kyma CLI

> [!WARNING]

> > This documentation is related with the Kyma CLI in version `v3` that is still in the prototyping stage. All commands are still considered alpha. Use it at your own risk.

Reviewer: ```suggestion
> This documentation applies to Kyma CLI version `v3`, which is still in the prototyping stage. All commands are still considered alpha. Use it at your own risk.
```
